tensorflow/tools/docker/Dockerfile.devel-cpu-mkl

```diff
@@ -1,73 +1,39 @@
-FROM ubuntu
+FROM tensorflow/tensorflow:latest-devel
 
 LABEL maintainer="Clayne Robison<clayne.b.robison@intel.com>"
 
-# It is possible to override these for releases.
+# These arguments are parameterized. Use --build-args to override.
 ARG TF_BRANCH=r1.4
 ARG BAZEL_VERSION=0.7.0
 ARG WHL_DIR=/whl
 
 RUN apt-get update && apt-get install -y --no-install-recommends \
-        build-essential \
-        curl \
-        git \
         golang \
-        libcurl3-dev \
-        libfreetype6-dev \
-        libzmq3-dev \
-        pkg-config \
-        python-dev \
-        python-pip \
-        rsync \
-        software-properties-common \
-        unzip \
-        zip \
-        zlib1g-dev \
-        openjdk-8-jdk \
-        openjdk-8-jre-headless \
-        wget \
-	locate \
-	vim \
+        vim \
+        emacs \
         && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
 RUN pip --no-cache-dir install --upgrade \
         pip setuptools
 
-RUN pip --no-cache-dir install \
-        ipykernel \
-        matplotlib \
-        numpy \
-        scipy \
-        sklearn \
-        pandas \
-        wheel \
-        && \
-    python -m ipykernel.kernelspec
-
-# Set up Bazel.
+RUN pip --no-cache-dir install wheel 
 
-# Running bazel inside a `docker build` command causes trouble, cf:
-#   https://github.com/bazelbuild/bazel/issues/134
-# The easiest solution is to set up a bazelrc file forcing --batch.
-RUN echo "startup --batch" >>/etc/bazel.bazelrc
-# Similarly, we need to workaround sandboxing issues:
-#   https://github.com/bazelbuild/bazel/issues/418
-RUN echo "build --spawn_strategy=standalone --genrule_strategy=standalone" \
-    >>/etc/bazel.bazelrc
+# Use the specified Bazel version
 WORKDIR /
-RUN mkdir /bazel && \
-    cd /bazel && \
-    wget --quiet https://github.com/bazelbuild/bazel/releases/download/$BAZEL_VERSION/bazel-$BAZEL_VERSION-installer-linux-x86_64.sh && \
-    wget --quiet https://raw.githubusercontent.com/bazelbuild/bazel/master/LICENSE && \
+RUN cd /bazel && \
+    rm -rf * && \
+    curl -H "User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/57.0.2987.133 Safari/537.36" -fSsL -O https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh && \
+    curl -H "User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/57.0.2987.133 Safari/537.36" -fSsL -o /bazel/LICENSE.txt https://raw.githubusercontent.com/bazelbuild/bazel/master/LICENSE && \
```

Comment:
why you need headers? looks strange...

Reply:
Just using the same command from the Tensorflow Dockerfile.devel that this is based on: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/docker/Dockerfile.devel#L64. Pulling a different rev of bazel.